web/src/components/nav_bar/nav_options.js

```diff
@@ -1,21 +1,34 @@
 import FeatureFlags from "../../infra/feature_flags";
 
 class NavOption {
-  constructor(displayName, url) {
+  constructor(displayName, url, isCurrentPageFunction) {
```

Comment:
set this to a default value of None

then 
`this.isCurrentPage = isCurrentPageFunction ?? () => window.location.hash === this.url`

Reply:
Noice, used the nullish coalescing operator.